audit/hashstructure_test.go

```diff
@@ -105,10 +106,13 @@ type TestSalter struct{}
 // storage instance.
 func (*TestSalter) Salt(ctx context.Context) (*salt.Salt, error) {
 	inmemStorage := &logical.InmemStorage{}
-	inmemStorage.Put(context.Background(), &logical.StorageEntry{
+	err := inmemStorage.Put(context.Background(), &logical.StorageEntry{
 		Key:   "salt",
 		Value: []byte("foo"),
 	})
+	if err != nil {
+		return nil, err
+	}
```

Comment:
You use `require.NoError(t, err)` for this in other tests. Maybe worth using here too?

<img src="https://media1.giphy.com/media/jPAdK8Nfzzwt2/giphy.gif"/>

Reply:
Yeah, I think I'm a bit stuck as this one is implementing the `Salter` interface's `Salt` method, so I can't adjust the func params to include `*testing.T`. Good shout though.